app/components/activities/list_item_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<li class="mb-10 ms-4">
+<li class="mb-4 ms-4 activity">
```

Comment:
**nitpick:** I don't really like random classes being added for testing purposes, is there another way you can do the test without adding this?

Reply:
Going to leave for now and we can update it later